plat/allwinner/common/sunxi_common.c

```diff
@@ -99,3 +101,58 @@ void sunxi_set_gpio_out(char port, int pin, bool level_high)
 			   0x7 << ((pin % 8) * 4),
 			   0x1 << ((pin % 8) * 4));
 }
+
+int platform_init_r_twi(uint16_t socid, bool use_i2c)
```

Comment:
Maybe `use_i2c` -> `use_rsb` would make it more clear what's going on. When I read this I was confused how you could have TWI without I²C :)